### DISCUSSION TITLE
add brave search engine to the mix

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#874] [Ideas]
### What feature would you like to see?

an option to have brave search in the setup screen

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/242_

**Comment by SerJaimeLannister** [comment]
Hey I would also like to state that we should add more search engines like startpage , qwant etc. as well while we are at it

I would also like there to show then in the first starting page when we open zen for the first time

Also one more minor critque which I am not sure deserves a whole issue is the fact that I don't think that google should be the default when clicked on next but rather we should have the user be forced to set their search engine themselves.

If they choose google , that's one step which is a minor inconvenience but it also slips the power of default , and people would have to "choose"

**Comment by ficd0** [comment]
I agree it would be nice to have brave as a default option (and possibly remove Google as the default!), but I want to point out that it is very easy to add a custom search engine as your default. All you need to do is go to a compliant search engine, like Brave, and right click the address bar, and there will be an option to "Add brave search". You can then go to the settings menu and set it as the default.

**Reply by SerJaimeLannister** [reply]
I do believe that this is a hassle considering unlike firefox , zen actually provides us an option in the beginning asking for our search engine 

So for zen this change of adding more search engines I suppose is a little easier than other firefox forks in my honest opinion

Also , Though something is easier doesn't imply that people would do it since somebody else mentioned to me that there was another way of adding search engine (where you enter search engine and click add and add search.brave.com/%s or something like this) and set it as default

there are numerous ways which are a little confusing & zen can really remove that confusion 

I think this method of showing multiple more search engines in start and also when clicking next , it doesn't automatically assume google search / will not go to next page unless you choose a search engine can be really good in the long run / also inspire other browsers in general
